catalog/ui/search-ui-app/src/main/resources/etc/org.codice.ddf.catalog.ui.transformer.blacklist.config

```diff
@@ -0,0 +1,5 @@
+blackListedMetacardTransformers=[ \
+]
+blackListedQueryResponseTransformers=[ \
+  "zipCompression", \
```

Comment:
Are we sure that `zipCompression` is not useful to anyone who wants to export a query response?